src/main/resources/application-dev.yml

```diff
@@ -10,7 +10,7 @@ spring:
     type: com.zaxxer.hikari.HikariDataSource
     url: jdbc:postgresql://localhost:5432/barter-bay-dev
     username: postgres
-    password: postgres
+    password: kolirzet09
```

Comment:
Line 19
Must be a standard "postgres" password, not a personal one.
Please correct it

Reply:
corrected